package.json

```diff
@@ -13,7 +13,7 @@
     "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ -- --inspect=0.0.0.0:9229 dist/server.js | bunyan -o short",
     "watch-sass": "npm run compile-sass -- --watch",
     "build": "npm run clean && npm run compile-sass && tsc && npm run copy-views",
-    "local:build": "npm install && npm run build && npm run test && ./scripts/start-local.sh && npm run int-test && ./scripts/stop-local.sh",
+    "local:build": "npm install && npm run build && npm run lint && npm run test && ./scripts/start-local.sh && npm run int-test && ./scripts/stop-local.sh",
```

Comment:
is this task designed to run on CI? Lint is already run by the circle config I think?

Reply:
no it wouldn't work on CI.  I just run it locally before I push.  I want this to be as close to what runs on CI as possible so I don't get failures in CI that could be caught earlier

Reply:
ok. Lint should run in pre-commit tbh, I'll check